### DISCUSSION TITLE
Failed to load component "544c1372-42df-47c3-94d6-017428cd2baf" (pnpSearchResultsWebPart) - The file was not found in the server.

### DISCUSSION THREAD

**Discussion by Ukiman1014** [microsoft-search/pnp-modern-search#2476] [Q&A] [answered]
**Version used**
4.8.0.0

**Describe the bug**
When adding the PnP Search Results web part, I immediately get the error.

**To Reproduce**
- Was running v4.7 fine but started having issues after attempting to update to 4.8.  I had uploaded 4.8 to the app store, but it was listed as a separate app, stating I needed to re-upload with the same name, so I deleted 4.8.
- PnP web parts started to disappear from pages or was getting Technical Error.  I then removed 4.7 from the site and re-added.  Did not help, so I removed it from the site and the org.  (It was only being used on one site.)
- Instead of trying to upgrade again, installed 4.8 to the org and the site.

**Expected behavior**
I expect to see the web part as I originally did with 4.7.

**Screenshots**
![PnP Search Results Error](https://user-images.githubusercontent.com/22643106/197028511-868cbfef-347d-46d0-821a-6a92375beb50.jpg)
![PnP Search Results Error - console](https://user-images.githubusercontent.com/22643106/197028626-a51e56b3-b650-4598-b7aa-1fe6bcf7523d.jpg)
![PnP Search Results Error - CDN](https://user-images.githubusercontent.com/22643106/197031048-ff499d0c-9c71-44d3-8bd1-a5b2509358f6.jpg)
![PnP Search Results Error - API](https://user-images.githubusercontent.com/22643106/197031733-7b1009cc-a492-462e-a049-46392261ae71.jpg)

**Desktop (please complete the following information):**
 - Edge Version 106.0.1370.47 (Official build) (64-bit)

**Additional context**
- I was also getting a Technical Error with v4.7 but I am not sure if it is the exact same error.
- I am not a tenant admin, so I did not allow API access for Teams-related connections, as I'm not using it for Teams at this point.
- Tried clearning the cache and cookies by opening developer tools > Application tab > Storage > "Clear site data" button.

**Comment by wobba** [comment, marked as answer]
Still an issue? And if you look thru the issue/forum list you see this popping up every time there is a new release. This is a SPFx CDN issue where it takes time for it to properly update. And usually resolves itself in a day or so.

**Reply by Ukiman1014** [reply]
I _almost_ thought it was still an issue.  I was off Friday so the CDN had three days to fix itself.  At first it was just the same.  I used dev tools to clear my cache and cookies for SharePoint... no difference.  _Held down control while refreshing the page_ and it was fixed!  I sometimes forget about holding control to make sure it's not using my local cache at all.  I thought clearing it would negate the need to do that, but I guess not.

Glad it resolved itself! 😁
